specification/gedcom-0-introduction.md

```diff
@@ -147,6 +147,7 @@ is shorthand for a URI beginning with the corresponding URI prefix
 | Short Prefix | URI Prefix                          |
 |:-------------|:------------------------------------|
 | `g7`         | `https://gedcom.io/terms/v7/`       |
+| `g7.1`       | `https://gedcom.io/terms/v7.1/`     |
```

Comment:
Should we use `71` or `7-1` or some other prefix that does not have a period so that the only period in URLs is used for file endings? Doesn't matter for most modern applicants, I suppose, but might for some older apps?